src/components/common/PageCareers.vue

```diff
@@ -2,7 +2,9 @@
   <TmPage data-title="Careers" hide-header>
     <div class="card">
       <h1>Careers</h1>
-      <h2 class="c5">Our Mission</h2>
+      <h2 class="c5">
```

Comment:
In a later PR we could turn these non app pages into markdown files?

Reply:
our new cms + website will handle this!